certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -3,676 +3,2194 @@
   "info": {
     "title": "Converter by Power2Apps",
     "description": "Enables conversion of different data formats from one to another.",
-    "version": "1.0",
     "contact": {
       "name": "Power2Apps Support",
       "url": "https://www.power2apps.de/kontakt",
       "email": "kontakt@power2apps.de"
-    }
-  },
-  "x-ms-connector-metadata": [
-    {
-      "propertyName": "Website",
-      "propertyValue": "https://www.power2apps.de"
-    },
-    {
-      "propertyName": "Privacy policy",
-      "propertyValue": "https://www.power2apps.de/datenschutz"
     },
-    {
-      "propertyName": "Categories",
-      "propertyValue": "Content and Files;Data"
-    }
-  ],
+    "version": "1.0.0"
+  },
   "host": "webapiconverterservice20211014190508.azurewebsites.net",
-  "basePath": "/Converter",
   "schemes": [
     "https"
   ],
+  "basePath": "/Converter",
   "paths": {
-    "/ConvertCsvdataToExcel": {
-      "post": {
+    "/GetWordSectionTypes": {
+      "get": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "GetWordSectionTypes",
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Success",
             "schema": {
-              "type": "object",
-              "properties": {
-                "FileContent": {
-                  "type": "string",
-                  "format": "byte",
-                  "x-ms-visibility": "important",
-                  "x-ms-summary": "File content",
-                  "description": "Content of the Excel file"
-                }
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ListOption"
               }
             }
           }
         },
-        "operationId": "ConvertCsvdataToExcel",
+        "x-ms-visibility": "internal",
```

Comment:
@stsiek The operation has been changed to hidden. This change breaks the designer for existing flows.

Reply:
we would like to set this action to deprecated, so it is set to "internal". we had understood that it can still be used for existing flows but is no longer visible when adding new actions. We also tested it and it works fine.

Reply:
@stsiek if you want to deprecate the current action, then please revert the changes and just add the deprecated property and set it to true. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/deprecate-operation).  Then you can add the internal action as a new action with a new operationId. 
Please let me know if you have any questions.

Reply:
thank you very much. That is good advice